### PR TITLE
Add 2023 Annual Report

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

File: content/en/project/status/2023MedleyAnnualReport.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* **Running in the browser:** Recently, we have been exploring the possibility of using recent advancements in the “Web Assembly” emulator to run Medley locally in a browser. There remain significant issues we will need to address, but this is a very promising direction.
+
+**Platforms, Releases, Builds, Automation, Installers**
+
+* **Support for SDL2:** We added to Medley support for SDL2 which can now be built from source. Further work is needed to provide appropriate binaries in the software releases. The SDL2 backend is useful running Medley on operating systems where X-Windows is not directly available, such as Windows, or which, being based on Wayland, may degrade the performance of X-Windows applications like Medley. Examples of the latter are Raspberry Pi OS Bookworm and later and ChromeOS.
```
Comment (pamoroso) [inline content/en/project/status/2023MedleyAnnualReport.md]: ...is useful running... -> is useful for running

File: content/en/project/status/2023MedleyAnnualReport.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+* **Content Organization and Accessibility:** In 2022, a first attempt at reorganizing the website was completed. Over the last year, we continued to revise and refine what and how to present material on our website. Changes are driven by new and improved content as well as improvements in our understanding of the story we need to tell. The result has been major improvements in website navigation. We continue to explore and work on new methods of enhancing our story telling. Work in progress includes a carousel on the landing page which will provide snapshots representing different facets of the Medley project.
```
Comment (pamoroso) [inline content/en/project/status/2023MedleyAnnualReport.md:58]: Is the carousel already active or in a branch waiting to be merged?

**Review by pamoroso — COMMENTED** [review]
I commented on a couple of minor issues but otherwise it looks good to me.

File: content/en/project/status/2023MedleyAnnualReport.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+We are continuing to spread information about the availability of Medley through technical presentations. This year, our team presented three talks on Medley Interlisp.
+
+* **BALISP:** In March 2023, we presented a discussion of the project’s efforts on restoring and modernizing Medley Interlisp to the Bay Area Lisp (BALISP) meetup group. [The slides are available](https://drive.google.com/file/d/1xpXSoEnc5PPnIa7BHcionBbc8v-Nxp7N/view?usp=sharing) and the talk was [recorded](https://www.youtube.com/watch?v=N1MobfEaoWY\&t=1s).
+* **SPN Idea Workshop:** On November 2, Larry Masinter participated in the Software Preservation Network’s Idea Workshop; we continue to pursue some academic projects in the software preservation community.
+* **Computer Conservation Society:** Steve Kaisler, a member of the Computer Conservation Society (CCS) in London, England, presented a talk entitled “Software Archeology: The Medley Interlisp Restoration Project” on November 16, 2023 in London, England. This talk discussed a brief history of Interlisp, some applications, and some issues and challenges in modernizing the Medley Interlips software. The talk was recorded and will likely be posted soon.
```
Comment (nbriggs) [inline content/en/project/status/2023MedleyAnnualReport.md]: Interlips should be Interlisp.

File: content/en/project/status/2023MedleyAnnualReport.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+* **UnixUtils:** In the last year we enhanced the set of tools that enables Medley to reach into its surrounding platform to accomplish tasks that are not available (and too extensive to implement) in Medley. As an example, the ShellBrowse function will open the specified URL within a browser running on the host platform. Similarly, ShellOpen can be used to open a PDF file using whatever PDF viewer is resident on the host platform.
+* **GITFNS (use of GIt and GitHub) extended**: Considerable effort went into the integration of Interlisp File Manager and GitHub, including a menu-driven interface to comparing Lisp source files on a function-by-function basis. This facility is crucial to the way the system manages the “residential style” development tools of Interlisp with GitHub.
```
Comment (nbriggs) [inline content/en/project/status/2023MedleyAnnualReport.md]: GIt should be Git

File: content/en/project/status/2023MedleyAnnualReport.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+#### **Project and Organization**
+
+As the work proceeds we expand our resources and adjust our processes to better achieve the project goals. In 2023 our team grew and we devised ways of keeping track of the work on the GitHub repositories.
+
+* Three new regular contributors joined the team, Matt Heffron, Paolo Amoroso and Andrew Sengul. Matt joined the team as a former Medley and Interlisp user and developer, and contributor to various projecta. Paolo exercised Medley as a user, reporting any bugs and issues that needed attention, and reviewed the existing documentation. Andrew, who came to the project as a Lisp developer, undertook a major redesign of the project site, restructuring it according to the project goals and making it easier to find the information users need, as well as clarifying and expanding the content. He also investigated how the development of Interlisp and other Lisps can proceed. As part of this work, for example, he started checking out the Medley test harness in view of using it again for testing system builds.
```
Comment (nbriggs) [inline content/en/project/status/2023MedleyAnnualReport.md]: projecta should be projects

File: content/en/project/status/2023MedleyAnnualReport.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
+# 2023 Medley Interlisp Project Annual Report
```
Comment (stumbo) [inline content/en/project/status/2023MedleyAnnualReport.md]: Replace with correct header in for Hugo:

```
---
title: 2023 Medley Interlisp Annual Report
weight: 1
type: docs
---
```

You will also want to change the weight on the 2022 Annual Report to another value, `9` is probably good to align it right before the 2021 Annual Report.

**Review by stumbo — CHANGES_REQUESTED** [review]

File: content/en/project/status/2023MedleyAnnualReport.md (hunk 2 above)
Comment (masinter) [reply content/en/project/status/2023MedleyAnnualReport.md:58]: I'm not sure. @phantomics?

**Review by pamoroso — COMMENTED** [review]
LGTM

**Review by stumbo — APPROVED** [review]

**Pull request merged**